src/extension/features/accounts/row-split-months/settings.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
note:  @joshmadewell  can you review this? I don't know exactly who to tag as reviewer, I looked into the last PRs' reviewers :)